src/components/DeleteMovieDialog/DeleteMovieDialog.jsx

```diff
@@ -22,4 +24,9 @@ const DeleteMovieDialog = ({ onClose, onDelete }) => {
     );
 };
 
-export default DeleteMovieDialog;
+const mapDispatchToProps = (dispatch) => ({
+    onClose: () => dispatch(movieActions.closeDeleteMovieDialog()),
+    onDelete: () => dispatch(movieActions.deleteMovie())
+})
+
+export default connect(null, mapDispatchToProps)(DeleteMovieDialog);
```

Comment:
I suggest to move connet and map to separate file as it's container 

it was ok for add modal because you actually just wrap, but here you have full modal implementation

Reply:
I agree with you, but i decided that i won't reuse the dialog and can use it as container. In future if i will need to have delete smth dialog i will move it in DeleteDialog, and also leave the DeleteMovieDialog as container. Is my logic correct?